specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -4265,7 +4396,11 @@
         },
         "vmCustomConfigurationUri": {
           "type": "string",
-          "description": "The uri to the storage blob containing scripts to be run on the virtual machine after provisioning."
+          "description": "The uri to the storage blob containing the arm template to be run on the virtual machine after provisioning."
+        },
+        "vmCustomConfigurationParameterUri": {
+          "type": "string",
+          "description": "The uri to the storage blob containing the parameter file to be run on the virtual machine after provisioning."
```

Comment:
I would like to have a comment to change ImageInfoProperties.properties.type enum values should be Gallery, CustomImage, and CustomVHD.  These are the values used in existing Create host pool ARM template shared in public repo ( https://github.com/Azure/RDS-Templates/blob/master/ARM-wvd-templates/CreateAndProvisionHostPool/CreateHostpoolTemplate.json#L197).  I confirmed with Vidya about this change and she agreed.  thanks.

Reply:
ok changing storageBlob to CustomVHD